### PR TITLE
Fix description for PyPI

### PR DESCRIPTION
**Pull request by di**
Update relative links to be absolute, and fix some typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,3 @@
-<head>
-  <base href="https://github.com/GoogleCloudPlatform/django-cloud-deploy/blob/master/"
```
Comment (brianquinlan) [inline README.md]: Does this not fix the problem with relative links?

File: README.md [hunk 2]
```diff
@@ -76,9 +71,9 @@ virtualenv -p python3 venv
 source venv/bin/activate
 ```
 
-Install Django-GKE in edit mode:
+Install `django-cloud-deploy` in edit mode:
```
Comment (brianquinlan) [inline README.md]:
```suggestion
Install **Django Deploy** in edit mode:
```

Use the official product name when possible.

File: README.md (hunk 1 above)
Comment (di) [reply README.md]: Unfortunately not, see https://pypi.org/project/django-cloud-deploy/

Comment (brianquinlan) [reply README.md]: OK, thanks!

**Pull request merged**